client/src/components/EditProfile/EditProfile.tsx

```diff
@@ -306,6 +465,50 @@ export default function EditProfile(): JSX.Element {
                 inputProps={{ maxLength: 40 }}
               />
             </Grid>
+
+            <Grid className={classes.formItem}>
+              <Typography className={classes.formLabel}>AVAILABILITY</Typography>
+              <Table>
+                <TableHead>
+                  <TableRow>
+                    <TableCell>AM/PM</TableCell>
+                    {DAYS_OF_THE_WEEK.map((day) => (
+                      <TableCell key={day} padding="none">
+                        {`${day.slice(0, 3)}.`}
+                      </TableCell>
+                    ))}
+                  </TableRow>
+                </TableHead>
+                <TableBody>
+                  <TableRow>
+                    <TableCell>AM</TableCell>
```

Comment:
Maybe we can mention the time slot here too? 
Eg: '9-5 AM' / '12-9 PM' ?

Reply:
We've decided to skip time slots for now. We are only going to set up the availability based on the days of the week. We might change that in the future as a stretch goal